spec/system/page_objects/components/navigation_menu/base.rb

```diff
@@ -0,0 +1,129 @@
+# frozen_string_literal: true
+
+module PageObjects
+  module Components
+    module NavigationMenu
+      class Base < PageObjects::Components::Base
```

Comment:
The methods here do not need to be reviewed, they are just a port from the old `PageObjects::Components::Sidebar`